lib/content-services/src/lib/tag/tags-creator/tags-creator.component.html

```diff
@@ -1,54 +1,41 @@
 <div class="adf-tags-creation">
+    <div class="adf-tag-name-field" *ngIf="(!tagNameControlVisible && tags.length) || tagNameControlVisible">
+        <mat-form-field class="adf-tag-search-field" appearance="fill">
+            <input #tagNameInput
+                    matInput 
+                    autocomplete="off"
+                    [formControl]="tagNameControl"
+                    (keyup.enter)="addTag()"
+                    aria-labelledby="adf-tag-name-input-label"
```

Comment:
that aria-labelledby seems incorrect. It has adf-tag-name-input-label but there is nowhere that id so that aria is linked to nothing.